src/Compilers/CSharp/Portable/Symbols/Synthesized/ReadOnlyListType/SynthesizedReadOnlyListTypeSymbol.cs

```diff
@@ -190,16 +226,18 @@
         }
 
         private readonly ModuleSymbol _containingModule;
+        private readonly bool _hasReadOnlyInterfaces;
```

Comment:
>_hasReadOnlyInterfaces [](http://example.com/codeflow?start=30&length=22)

Consider removing the field if it's not needed (if we can simply use the constructor parameter).